runtime/Go/antlr/atn_config.go

```diff
@@ -285,6 +285,24 @@ func (l *LexerATNConfig) equals(other interface{}) bool {
 	return l.BaseATNConfig.equals(othert.BaseATNConfig)
 }
 
+func (l *LexerATNConfig) HashCode() int {
+	var f int
+	if l.passedThroughNonGreedyDecision {
```

Comment:
Again, can you point me at a style guide. Vscode fmt's the code on every save. As for white space between lines inside blocks, imo they should be used really sparingly. Is there a style guide for the Java antlr code?